### ISSUE TITLE
device.getDeviceInfo and modem.getDeviceInfo return different structures

### ISSUE THREAD

**Issue by PaulWieland** [apearson/insteon-plm#40]
These two methods return two different objects. Is there a reason not to have the `device.getDeviceInfo` call the static method `PowerLincModem.getDeviceInfo`?

**Comment by apearson** [comment]
It looks like the Modem function gets the device db info from the cat and subcat.

whereas 

the device class does a id request to get cat, subcat, and hardware on device.

I agree these should be named differently.

**Comment by PaulWieland** [comment]
`PowerLincModem.getDeviceInfo` is called by `PowerLincModem.queryDeviceInfo`

The `PowerLincModem.queryDeviceInfo` sends the ID request to the device. What i'm proposing is that we just re-use these static methods provided by the PLM class in the device class so that `device.getDeviceInfo` returns the same object as `PowerLincModem.getDeviceInfo`.

**Comment by apearson** [comment]
`device.getDeviceInfo` also returns the hardware and firmware version of the particular device but I  get where you're coming from and I'm working on a unification of the names

**Comment by apearson** [comment]
This is corrected (c2d81c7) with the following being the resolved api.  More work should go into this to use the static methods but I'm leaving that for an API clean up

```
console.log('Device');
console.log(await sunroomD.getDeviceInfo())
console.log(await sunroomD.getFullDeviceInfo())

console.log('Modem');
console.log(await modem.queryDeviceInfo(sunroom));
console.log(await modem.queryFullDeviceInfo(sunroom));
```
